packages/sindarian-i18n-cli/src/validator.test.ts

```
@@ -0,0 +1,161 @@
+      ['a.ts', [makeMessage({ id: '', file: 'a.ts', line: 5, col: 3 })]]
+    ])
+
+    const validation = validate(result)
+    expect(validation.errorCount).toBe(1)
+    expect(validation.issues[0]).toMatchObject({
+      severity: 'error',
+      file: 'a.ts',
+      line: 5,
+      col: 3
+    })
+    expect(validation.issues[0].message).toMatch(/missing id/i)
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)